src/poprox_concepts/domain/newsletter.py

```diff
@@ -12,3 +13,4 @@ class Newsletter(BaseModel):
     articles: list[Article]
     subject: str
     body_html: str
+    created_at: datetime | None = None
```

Comment:
My only question is whether we want this to default to `None` or to the current time. I can see pros and cons both ways.

I'm guessing the current use-case is exporting data, in which case None is a better default. right?

Reply:
Yeah, that's the current use case. If we decide later to default it to the current time, we can do that too.